metrics.md

```diff
@@ -0,0 +1,47 @@
+ 
+#### Gluster prometheus end point schema: 
+
+#### Volume Metrics : 
+
+|Gluster Promethus schema| example| codes |
+|--------|-----------|---|
+|gluster_peer_status |	gluster_peer_status{nodeid="xxxxx"} 1 |	1 - connected, 0 - disconnected	
+|gluster_volume_status|	gluster_volume_status{volume="xxx"} 1	| 1 - started, 0 - stopped, 2 - starting, 3 - stopping	
+|gluster_volume_fop_max_latency	| gluster_volume_max_latency{volume=""xxx"", brick_path=""xxxx"",fop=""xxx""}  800970.00| 	
+|gluster_volume_fop_min_latency	|gluster_volume_fop_min_latency{volume="xxx", brick_path="xxx", fop="xxx"} 21.00	|
+|gluster_volume_fop_avg_latency|	gluster_volume_fop_avg_latency{volume="xxx", brick_path="xxx", fop="xxx"} 1132.00|	
+|gluster_volume_fop_calls |gluster_volume_fop_calls{volume="xxx". brick_path="xxx", fop="xxx"} 13	|
+|gluster_volume_fop_percentage_latency| gluster_volume_fop_percentage_latency(volume="xxx", brick_path="xxx", fop="xxx"} 13	|
+|gluster_volume_total_bytes| gluster_volume_total_bytes{volume="xxx"}  104857600	|
+|gluster_volume_used_bytes| gluster_volume_used_bytes{volume="xxx"}  4857600	|
+|gluster_volume_available_bytes|	gluster_volume_available_bytes{volume="xxx"} 9856780	|
+
+#### Brick Metrics:
+|Gluster Promethus schema| example
+|--------|-----------|
+| gluster_brick_inodes_used |	gluster_brick_inodes_used{volume="xxx", "brick_path"="/mnt/xxx", nodeid="xxxxx"} 100	|
+| gluster_brick_inodes_total |	gluster_brick_inodes_total{volume="xxx", "brick_path"="/mnt/xxx", nodeid="xxxxx"} 200 |	
+| gluster_brick_inodes_available | gluster_brick_inodes_available{volume="xxx", "brick"="/mnt/xxx", nodeid="xxxxx"} 100 |
+| gluster_brick_size_bytes | gluster_brick_size_bytes{volume="xxx", brick_path="xxxx", nodeid="xxx"} 1048576	|
+| gluster_brick_used_bytes|	gluster_brick_used_bytes{volume="xxx", brick_path="xxxx", nodeid="xxx"} 1048	|
+| gluster_brick_available_bytes |	gluster_brick_available_bytes{volume="xxx", brick_path="xxx", nodeid="xxx"} 1047576	|
+| gluster_disk_read_ios | gluster_disk_read_ios{brick_path="xxx",device="xxx",disk="vda",host="xxx",id="xxx",instance="xxx:8080",subvolume="xxx",volume="xxx"} 0
```

Comment:
is this volume id or peer id?

Reply:
Yes when we say `id` its `brickid` actually.